metrics/stability/degradation_analysis.py

```diff
@@ -0,0 +1,83 @@
+"""
```

Comment:
Shouldn't this file be under `metrics/utils/` to match the existing folder structure of the Python project? It seems that all the other metrics (cluster, storage) where committed there. If we want to have a folder per metric type, maybe the others should be moved into their own folder (cc @anjagruenheid )? WDYT?